examples/discovery-search-app/package.json

```diff
@@ -23,8 +22,8 @@
   },
   "dependencies": {
     "@carbon/icons": "^10.5.0",
-    "@ibm-watson/discovery-react-components": "file:../../packages/discovery-react-components",
-    "@ibm-watson/discovery-styles": "file:../../packages/discovery-styles",
+    "@ibm-watson/discovery-react-components": "^1.5.0-beta.0",
+    "@ibm-watson/discovery-styles": "^1.4.1-beta.1",
```

Comment:
Can't use `workspace:` or `portal:`, since those aren't supported by lerna. `file:` was also giving me issues, resulting in a different `yarn.lock` on Travis vs locally. In the end, just added the current versions -- yarn will actually map these on to the local workspace folder, since the versions are compatible.